pkgs/development/libraries/LAStools/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, cmake }:
+stdenv.mkDerivation rec {
+
+  pname = "LAStools-${version}";
```

Comment:
`version` should not be apart of `pname`.
```suggestion
  pname = "LAStools";
```

Reply:
fixed in 3681d4f280cfdc22c3c49f4f1e6d714e0881783d